public/e2e-tests/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["**/*.ts"]
+  "compilerOptions": {
+    "types": ["cypress"]
+  },
+  "include": ["**/*.ts", "../../packages/grafana-e2e/cypress/support/index.d.ts"]
```

Comment:
This is so that we can get correct typings when writing tests in main Grafana repo.